src/utils/image.ts

```diff
@@ -78,32 +78,41 @@ export const getFirebaseStorageResizedImage = (
 // @see https://crypto.stackexchange.com/questions/8533/why-are-bitwise-rotations-used-in-cryptography/8534#8534
 const DIFFUSION_PRIME = 31;
 
-type DetermineAvatar = (options?: {
+type DetermineAvatarProps = {
   avatars?: string[];
   email?: string;
   index?: number;
   partyName?: string;
   pictureUrl?: string;
   userInfo?: firebase.UserInfo;
   user?: User;
-}) => string;
+};
+
+type DetermineAvatarFunc = (
```

Comment:
No need of `Func` suffix. 

`DetermineAvatar` vs `determineAvatar` is the convention used elsewhere

Reply:
done: https://github.com/sparkletown/sparkle/pull/2760/commits/9cc6e165b25244d4599206263d7e1e47ab5f583f